projects/2023-NH/analysis.script_hadacek.R

```diff
@@ -110,42 +107,63 @@ jointdataset <- merge(data_self, peer_group, by = 'ID')
 jointdataset <- jointdataset %>% 
   dplyr::mutate(consci_dif = (consci_peer - consci_self))
 
+# DATA VISUALIZATION ####
+
+sconsci_dat_descr <- data.frame(describe(jointdataset$consci_self), 
+                                row.names = "sconsci_dat_descr")
+pconsci_dat_descr <- data.frame(describe(jointdataset$consci_peer), 
+                                row.names = "pconsci_dat_descr")
+pid_dat_descr <- data.frame(describe(jointdataset$pid), 
+                            row.names = "pid_dat_descr")
+pil_dat_descr <- data.frame(describe(jointdataset$pil), 
+                            row.names = "pil_data_descr")
+neuro_dat_descr <- data.frame(describe(jointdataset$neuro), 
+                              row.names = "neuro_data_descr")
+withdraw_dat_descr <- data.frame(describe(jointdataset$withdraw), 
+                                 row.names = "withdraw_dat_descr")
+
+dat.df <- t(rbind.data.frame(sconsci_dat_descr, pconsci_dat_descr, 
+                             pid_dat_descr, pil_dat_descr,neuro_dat_descr, 
+                             withdraw_dat_descr))
+
+corr.test(jointdataset$consci_peer, jointdataset$consci_self)
```

Comment:
cor.test() might be preferable (from the native stats package). it shows more information by default, including t-value and confidence interval, and more decimal points